helm/external-dns-app/values.schema.json

```diff
@@ -11,9 +11,6 @@
                 "access": {
                     "type": "string"
                 },
-                "accountID": {
```

Comment:
Can you please add that back? Thank you!

Reply:
Good catch. Thanks!